forem/forem
# Fixed display of CoC/Terms in/onboarding

## What type of PR is this?


## Description



## Related Tickets & Documents


- Related Issue #19395 

## QA Instructions, Screenshots, Recordings

1. Login and go to `/onboarding` and click `Code of Conduct` and `Terms and Conditions`
3. The content fills the available space, accounting for padding

Expected behavior:
![Screenshot 2023-04-29 at 7 02 57 PM](https://user-images.githubusercontent.com/74767965/235299303-582c410e-f1b9-41e6-8afd-3dcd586de358.jpg)


### UI accessibility concerns?

More readable, the content would not been cut as half.

## Added/updated tests?

- [x] No, and this is why: I think that just make height to fix the outside box, so we don't need to add test for that.
